src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html

```diff
@@ -73,7 +73,7 @@
                  id="is-regex">
           <label for="is-regex"
                  class="custom-control-label"
-                 i18n>Use regular expression</label>
+                 i18n>Enable</label>
```

Comment:
Why is this change?

Reply:
Yes, "Use regular expression" makes more sense. I have changed it back to it.